trunk/src/app/srs_app_http_stream.cpp

```diff
@@ -711,10 +714,16 @@ srs_error_t SrsLiveStream::do_serve_http(ISrsHttpResponseWriter* w, ISrsHttpMess
     // Enter chunked mode, because we didn't set the content-length.
     w->write_header(SRS_CONSTS_HTTP_OK);
 
-    SrsSharedPtr<SrsLiveSource> live_source = _srs_sources->fetch(req);
-    if (!live_source.get()) {
-        return srs_error_new(ERROR_NO_SOURCE, "no source for %s", req->get_stream_url().c_str());
+    SrsSharedPtr<SrsLiveSource> live_source;
```

Comment:
conflict with #4166 ?

Reply:
Yep, I will fix the conflict after merging one of these.

Reply:
Fixed.